### ISSUE TITLE
Date formatting utils

### ISSUE THREAD

**Issue by Windvis** [appuniversum/ember-appuniversum#201]
All projects have a need to format dates before showing the data to the users. This is usually done by using ember-moment`s `{{format-date}}` helper with a custom format, or with custom helpers that each project implement themselves.

To reduce the custom code, and make it consistent across projects it might be better to make this part of ember-appuniversum? It's not strictly related to appuniversum itself, so a dedicated addon could also work.

**Comment by Dietr** [comment]
Yes, we also really need some guidelines in the documentation here. Not only for dates but other things like button labels (add, delete, ...), page titles, general copy. I prefer to do this in ember-appuniversum. 

From what I see in most projects `21/02/2022` seems the most unambiguous. Using dots as separators seems more fitted for numbers like `10.000,00`.

**Comment by Windvis** [comment]
Sounds good!

I think the format needs to be decided by someone at ABB though. Felix seemed hesitant to switch to the `/` version since it might make certain users unhappy, which makes me feel it's a more sensitive issue than I expected :smile:.

I also found this: https://taaladvies.net/datumnotatie/ which mentions the following:

> De notatie met streepjes is vooral in Nederland gebruikelijk. De notatie met punten is de schrijfwijze die in België is vastgelegd in de norm NBN Z 01-002 Indelen en opmaken van documenten (2002) van het Bureau voor Normalisatie (NBN).

Which makes it sound like the `.` version is somewhat official?

**Comment by Windvis** [comment]
The decision was made to use the `DD-MM-YYYY` format so this is now unblocked.

**Comment by Windvis** [comment]
I'm still not 100% sure what this should look like.

Do we simply create a `au-format-date` helper (and formDate util for JS contexts) that expects a date and outputs `DD-MM-YYYY`? I know some projects also use a longer version but I'm not sure if we should support that here.
